### PR TITLE
56-create-collections-pages

### LINKED ISSUE #56: Create Collections pages
e.g. https://www.24life.com/collections/recipes/

Can re-use the `Article List ` block as on https://main--24life--hlxsites.hlx.page/author/trish-shallest
Just need to add another configuration option to filter by collection, instead of author-id.

### PR DESCRIPTION
**Pull request by iamitshri**
- article list block used for fetching collection
- hero block modified 

Fix #56 #23 

Test URLs:
- Before: https://main--24life--hlxsites.hlx.page/
- After: 

- https://56-create-collections-pages--24life--hlxsites.hlx.page/collections/workouts
- https://56-create-collections-pages--24life--hlxsites.hlx.page/collections/success-stories
- https://56-create-collections-pages--24life--hlxsites.hlx.page/collections/mental-hacks
- https://56-create-collections-pages--24life--hlxsites.hlx.page/collections/feel-better

### CODE REVIEW

File: blocks/card/card.css [hunk 1]
```diff
@@ -114,7 +114,7 @@
 
 /* on smaller screens, show everything in a single column. */
 @media (max-width: 900px) {
-  .section.card-container {
+  .card-container.three-columns{
```
Comment (synox) [inline blocks/card/card.css:117]: Now this is only applied to 3-col layouts, but not to 2-col layout any more. See https://56-create-collections-pages--24life--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/block-library/blocks/card&index=1

You could comma-separate multiple selectors.

Comment (iamitshri) [reply blocks/card/card.css:117]: oops I will fix this right away

File: blocks/collection-hero/collection-hero.css [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    letter-spacing: 1px;
+  }
+
+  .collection-hero p.hero-collection-subtitle{
+    font-size: 16px;
+    letter-spacing: 1px;
+    line-height: 28px;
+  }
+
+@media (min-width: 1200px){
```
Comment (rrusher) [inline blocks/collection-hero/collection-hero.css:56]: There are two media queries for 1200px. One is specifically for screens (i.e. not applied when printing) and the other applies regardless of the device. Is this intentional?

**Review by rrusher — APPROVED** [review]

File: blocks/collection-hero/collection-hero.css (hunk 2 above)
Comment (iamitshri) [reply blocks/collection-hero/collection-hero.css:56]: Not intentional.  I will fix it by adding `@media screen and `

**Pull request merged**